doc/getting/Releases.rst

```diff
@@ -151,9 +151,7 @@ On Windows
 
    All Windows command line instructions are intended for :program:`Windows PowerShell`,
    if not marked otherwise. So executing the following instructions in Windows
-   Command Prompt (:program:`cmd.exe`) won't function or result in errors! See
-   the :ref:`Requirements section <GETTING:Require>` on where to
-   download or update PowerShell.
+   Command Prompt (:program:`cmd.exe`) won't function or will result in errors!
```

Comment:
Why was the link removed?

Reply:
Oh, I see. This text snippet was copied from the PoC-Library's documentation. Therein it links to [Requirements](http://poc-library.readthedocs.io/en/release/UsingPoC/Requirements.html#windows-specific-requirements) and lists 2 working PowerShell versions. PCs with Windows 10 have already PowerShell 5.

I think we need a documentation section explaining how to get PowerShell on Windows 7/8.1.

Reply:
Windows 7 comes with PowerShell. Do you want a section on how to upgrade it to PowerShell 5?

Reply:
@Paebbels see above comment?

Reply:
@gritbub Yes an upgrade / installation section would be good. The PowerShell in Windows 7 is only version 2. The minimum requirement is 4.0, But there is no need to explain that installation, when the world - as in Windows 10 - is using 5.0.

Reply:
Done [here](https://github.com/ghdl/ghdl/pull/591/commits/44772d3c1c421df84c1642d2331b8c65c395094b)